code/datums/martial/cqc.dm

```diff
@@ -80,9 +80,21 @@
 		return TRUE
 
 /datum/martial_art/cqc/proc/Kick(mob/living/attacker, mob/living/defender)
-	if(!can_use(attacker))
+	if(!can_use(attacker) || defender.stat == DEAD)
 		return FALSE
-	if(!defender.stat || !defender.IsParalyzed())
+
+	var/helmet_protection = defender.run_armor_check(BODY_ZONE_HEAD, MELEE)
+	var/knockout_prob = rand(-15,15) + defender.getStaminaLoss() - helmet_protection
+	if(defender.body_position == LYING_DOWN && !defender.IsUnconscious() && prob(knockout_prob))
+		log_combat(attacker, defender, "knocked out (Head kick)(CQC)")
+		defender.visible_message(span_danger("[attacker] kicks [defender]'s head, knocking [defender.p_them()] out!"), \
+						span_userdanger("You're knocked unconscious by [attacker]!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), null, attacker)
+		to_chat(attacker, span_danger("You kick [defender]'s head, knocking [defender.p_them()] out!"))
+		playsound(get_turf(attacker), 'sound/weapons/genhit1.ogg', 50, TRUE, -1)
+		defender.apply_effect(20 SECONDS, EFFECT_KNOCKDOWN, helmet_protection)
+		defender.SetUnconscious(10 SECONDS)
+		defender.adjustOrganLoss(ORGAN_SLOT_BRAIN, 15, 150)
+	else
```

Comment:
The else clause here results in the head kick doing absolutely no brute damage or the knockback. That doesn't really make any sense for a critical hit effect. This should be the original order of operations and use the probability there to prevent any weird no-damage harm hits.

Reply:
There was no damage done for the knockout blow in the original code. I think it's fine considering you are causing a severe status effect and organ damage to the target.

Reply:
Ah, yep, missed the parent call. Fair enough.